src/main/java/io/curity/identityserver/plugin/authenticationaction/debug/attribute/representations/IndexRepresentationFunction.java

```diff
@@ -0,0 +1,36 @@
+package io.curity.identityserver.plugin.authenticationaction.debug.attribute.representations;
+
+import se.curity.identityserver.sdk.haapi.HaapiContract;
+import se.curity.identityserver.sdk.haapi.Message;
+import se.curity.identityserver.sdk.haapi.RepresentationFactory;
+import se.curity.identityserver.sdk.haapi.RepresentationFunction;
+import se.curity.identityserver.sdk.haapi.RepresentationModel;
+import se.curity.identityserver.sdk.http.HttpMethod;
+import se.curity.identityserver.sdk.web.Representation;
+
+import java.net.URI;
+
+public final class IndexRepresentationFunction implements RepresentationFunction
+{
+    private static final Message MSG_CONTINUE = Message.ofLiteral("Continue");
+    private static final Message MSG_SUBJECT_ATTRS = Message.ofLiteral("Subject Attributes");
+    private static final Message MSG_CONTEXT_ATTRS = Message.ofLiteral("Context Attributes");
+
+    @Override
+    public Representation apply(RepresentationModel model, RepresentationFactory factory)
+    {
+        return factory.newAuthenticationStep(step -> {
+            step.addMessage(Message.ofLiteral("Subject: " + model.getString("_subject")), HaapiContract.MessageClasses.HEADING);
```

Comment:
This feels wrong as it's not translating the `Subject: ` "header".... can't we use a parameterised message instead?

Reply:
The current HTML templates don't translate it either. Since this is an authentication action that we explicitly say shouldn't be used in Production, I guess this is OK.